### DISCUSSION TITLE
Change Dropdown Language

### DISCUSSION THREAD

**Discussion by Aledcode** [GreaterWMS/GreaterWMS#229] [Announcements]
Hi, how to change the dropdown text languange, 
drop down not appear in en-us event the language already change
![image](https://user-images.githubusercontent.com/110827613/189950327-1d69a325-7136-4234-b9b0-57fe02c894da.png)

**Comment by Singosgu** [comment]
Yup, We have fix it

it seems staff_type_list just support CN , we change it support others

**Reply by Aledcode** [reply]
So how must I to do, mean which file that need I change/update?

**Comment by Singosgu** [comment]
templates/src/pages/staff/stafflist.vue

don't forget quasar build

**Reply by Aledcode** [reply]
I try, but seems like not working
![image](https://user-images.githubusercontent.com/110827613/190298917-56b978c1-5bcf-428d-b55f-640549f26224.png)

**Reply by Singosgu** [reply]
reinstall quasar -g & template yarn install

**Comment by Aledcode** [comment]
So what must I todo?
Means on which file need update/ change with the latest one?
